foursight_core/run_result.py

```diff
@@ -25,6 +30,13 @@ def __init__(self, connections, name):
         self.extension = ".json"
         self.kwargs = {}
 
+    @staticmethod
```

Comment:
Maybe move this to top-level so it doesn't have to be called as `self.dump_json(...)` when it has nothing to do with `self`. Maybe we should have some functions like this in `dcicutils` for use elsewhere. I might call it `json_str` or `as_json`. Note that `dumps` is so-named because it's the string version of `dump` where `dump` writes to a file. This isn't involving files, but is honoring the "dump" (write to file) part of the name "dumps" but not the "s" part.  The only issue with making it a general purpose function is the `default=str`, but maybe if such a utility took a `strict=` argument that defaulted to `False` (which would add `default=str`. It could still take the `indent=` argument, too, but default it compatibly with whatever `json.dumps` uses (probably `None`).

Reply:
I'm fine with refactoring this to utils eventually. Yes the `default` parameter here is key to coerce objects to string form when storing responses from AWS, which commonly include `datetime` objects (that are not JSON serializable). I did make some small changes to address this but this fix should fix it globally.